src/atoms/Loader/Atom/index.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
```

Comment:
I'm not really getting the ideea behind the folder structure here. The loader should simply a single HOC. We should discuss this.